openwisp_network_topology/management/commands/__init__.py

```diff
@@ -1,12 +1,172 @@
-import swapper
```

Comment:
I would leave __init__ empty and move this code to its own file, so it will be easier to find it and remove it in the future.